### PR TITLE
@orta=>admin option + complete ib work for master view

### PR DESCRIPTION
**Pull request by sarahscott**
- [x] interface for master view of new settings
- [x] admin toggle

iPad-only for now

:dancer: 

simulator screen shot sep 18 2015 3 23 49 pm

### CODE REVIEW

**Comment by sarahscott** [comment]
:green_apple:

**Comment by sarahscott** [comment]
unless you want me to write up the view controllers in swift before merging

File: Classes/Controllers/Settings/ARLabSettings.storyboard [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+        </mutableArray>
+        <mutableArray key="ITC_Avant_Garde_Gothic__Demi.ttf">
+            <string>AvantGardeGothicITCW01Dm</string>
+            <string>AvantGardeGothicITCW01Dm</string>
+            <string>AvantGardeGothicITCW01Dm</string>
+            <string>AvantGardeGothicITCW01Dm</string>
+            <string>AvantGardeGothicITCW01Dm</string>
+            <string>AvantGardeGothicITCW01Dm</string>
+            <string>AvantGardeGothicITCW01Dm</string>
+            <string>AvantGardeGothicITCW01Dm</string>
```
Comment (orta) [inline Classes/Controllers/Settings/ARLabSettings.storyboard]: Giphy

File: Classes/Controllers/Settings/ARLabSettingsMasterViewController.m [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#import "ARLabSettingsMasterViewController.h"
+#import "AROptions.h"
+#import "ARLabSettingsNavController.h"
+
+
+@interface ARLabSettingsMasterViewController ()
+@property (weak, nonatomic) IBOutlet UIButton *settingsButton;
+@property (weak, nonatomic) IBOutlet UIButton *ogSettingsButton;
```
Comment (orta) [inline Classes/Controllers/Settings/ARLabSettingsMasterViewController.m]: hah, `ogSettingsButton`

**Comment by orta** [comment]
This looks good to me :+1:

**Pull request merged**